kube/pod.go

```diff
@@ -110,8 +110,11 @@ func getContainerImageName(role *model.Role, settings *ExportSettings) (string,
 	if err != nil {
 		return "", err
 	}
-	imageName := builder.GetRoleDevImageName(settings.Registry, settings.Organization, settings.Repository, role, devVersion)
-	return imageName, nil
+	imageName := builder.GetRoleDevImageName("", "", settings.Repository, role, devVersion)
+	value := "{{ default \"docker.io\" .Values.kube.registry -}}/"
```

Comment:
Why don't we put the `docker.io` default value into `values.yaml` instead?

I don't have strong feelings either way, but my gut instinct is to define a default value in one place, and not repeat it wherever it is used. But then we have the `require` mechanism, which can't be specified in `values.yaml`, so maybe it is better the way it is? Can't make up my mind. 😄